filament/include/filament/RenderableManager.h

```diff
@@ -340,6 +342,50 @@ class UTILS_PUBLIC RenderableManager : public FilamentAPI {
         Builder& skinning(size_t boneCount, Bone const* bones) noexcept; //!< \overload
         Builder& skinning(size_t boneCount) noexcept; //!< \overload
 
+        /**
+         * Define bone indices and weights for vertex skinning. The first pair value defines index
```

Comment:
Maybe specify that a "pair" is a `float2` and the bone index is interpreted as `unsigned int(pair.x)`.

Reply:
good point, fixed